sherpa/plot/testing.py

```diff
@@ -0,0 +1,116 @@
+#
```

Comment:
So, I have stared deep into the heart of our build system and it has stared back at me. And won. So I am always "interested" in things that could have some "interesting" consequences for our builds. I believe (too lazy to check) that we include the `test_*py` files in the MANIFEST (either directly, indirectly, or magically) and maybe we get this file for free (e.g. we - or whatever system decides, uses 'test*py' or '*py' as the regexp). Do you know how this works?

Reply:
This is not a "test_.py" file, it's part of the real deal; it just happens to have a filename that also starts with T E S T as first four letters. If it was not included, the tests woudl fail, but I've neve added other source files (e.g. `bokeh_backend.py`) manually to MANIFEST and we've always gotten them, so - while I don't understand our build system fully - I'm ptryy sure this is save?

Reply:
For some reason I thought it was `sherpa/plot/tests/testing.py`...

Reply:
I had it there originally, but I had trouble importing it. Maybe because some test dirs don't have `__init__.py`? We can move it later if you want?